embulk-output-bigquery.gemspec

```
@@ -1,6 +1,6 @@
 Gem::Specification.new do |spec|
   spec.name          = "embulk-output-bigquery"
-  spec.version       = "0.6.13"
+  spec.version       = "0.6.14"
```

bugfixじゃないので、 0.7.0かな
```suggestion
  spec.version       = "0.7.0"
```

---

承知しました！